tests/files/environments/development/us-east-1.yml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  parameters:
+    CidrBlock: '10.0'
+stacks:
+  - name: Test-Dev-VPC
+    template_name: VPC
+    parameters:
+      Name: "TestVPC"
+
+  - name: Test-Dev-SecurityGroup
+    template_name: SecurityGroup
+    parameters:
+      VpcId:
+        Stack: Test-VPC
+        Output: VpcId
```

Comment:
Yeah I'd be happy with something like that, and it's my ideal world too. I just didn't have time for that substantial a change and already get value out of the old way plus what are basically environment namespaces.